File: app/src/main/java/com/qwert2603/vkmessagestat/base/recyclerview/ClickCallbacks.java
package com.qwert2603.vkmessagestat.base.recyclerview;

/**
 * Callback для нажатия на элемент.
 */
public interface ClickCallbacks {
    /**
     * Нажатие на элемент.
     *
     * @param position позиция нажатого элемента
     */
    void onItemClicked(int position);
}

File: app/src/main/java/com/qwert2603/vkmessagestat/base/recyclerview/ClickCallbacks.kt
package com.qwert2603.vkmessagestat.base.recyclerview

/**
 * Callback для нажатия на элемент.
 */
interface ClickCallbacks {
    /**
     * Нажатие на элемент.
     *
     * @param position позиция нажатого элемента
     */
    fun onItemClicked(position: Int)
}
